src/containers/Storage/Storage.js

```diff
@@ -159,8 +170,20 @@ class Storage extends React.Component {
 
     renderLoader() {
         return (
-            <div className={b('loader')}>
-                <Loader size="m" />
+            <div className={b('skeleton')}>
```

Comment:
We have a lot of tables in this apps that may need a Skeleton in the near future. I suggest to make Table Skeleton as a separate component.